components/scoreHandler.ts

```diff
@@ -1144,6 +1206,10 @@ export async function getMissionEndData(
         },
     }
 
+    if (isDryRun) {
+        return result
+    }
```

Comment:
Below this is only logic for committing scores & handling livesplit updates, which we can just ignore.